DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -117,11 +122,44 @@ public async Task AddEventAsync(string name, string description, DateTime time)
         }
 
         public async Task EditEventAsync(
-            int key,
+            int eventKey,
             string? description = null,
             string? name = null,
-            DateTime? time = null)
+            DateTime? time = null
+        )
         {
+            var rowNumber = await GetEventRowNumber(eventKey);
+
+            var data = new List<ValueRange>();
+
+            if (name != null)
+            {
+                data.Add(MakeCellUpdate($"EventsMetadata!B{rowNumber}", name));
```

Comment:
Is it worth storing "B", "C" and "D" as well named constants making it clear they're the column names for the name, description and time respectively (as I assume they are)? 🙂

Reply:
Yes that's a good idea! Done that 02e76dd 🙂